CHANGES.rst

```diff
@@ -2,11 +2,18 @@
 CHANGES
 =======
 
-3.6.5 (unreleased)
+4.0.0 (unreleased)
 ------------------
 
-- Nothing changed yet.
+- Add support for Python 3.5, 3.6 and PyPy.
 
+- Replace dependency on ``ZODB3`` with ``BTrees`` and ``persistent``.
+
+- Drop test dependency on ``zope.app.testing``.
+
+- The synchronization view now uses Python's built-in transport for
+  handling Basic Authentication. This means that the username and
+  password should not contain a ``:``.
```

Comment:
Any compatibility concerns we might have here would be on the Zope server side, in the Authorization header parser, if it happens to use `.split(':')` instead of `.split(':', 1)`.